nxdrive/gui/application.py

```diff
@@ -1669,14 +1669,20 @@ def refresh_transfers(self, dao: EngineDAO) -> None:
     def refresh_direct_transfer_items(self, dao: EngineDAO) -> None:
         transfers = self.api.get_direct_transfer_items(dao)
         if transfers != self.direct_transfer_model.items:
-            self.direct_transfer_model.set_items(transfers)
+            if not self.direct_transfer_model.items:
```

Comment:
Could you store `self.direct_transfer_model.items` into a var to prevent double lookup? Same for `refresh_active_sessions_items()`.